src/discordtipbot/discordbot.cr

```diff
@@ -128,11 +128,9 @@ class DiscordBot
   end
 
   private def dm_deposit(user : UInt64)
-    begin
-      @bot.create_message(@cache.resolve_dm_channel(user), "Your deposit just went through! Remember: Deposit Addresses are *one-time* use only so you'll have to generate a new address for your next deposit!\n*#{TERMS}")
-    rescue
-      @log.info("#{@config.coinname_short}: Failed to contact #{user} with deposit notification")
-    end
+    @bot.create_message(@cache.resolve_dm_channel(user), "Your deposit just went through! Remember: Deposit Addresses are *one-time* use only so you'll have to generate a new address for your next deposit!\n*#{TERMS}")
+  rescue
```

Comment:
While the cases in which the above code would fail are usually pretty obvious, you should probably capture the exception and print it in the log line below.